src/components/m-button/mixins/_button.scss

```diff
@@ -347,19 +344,23 @@
 }
 
 @mixin button--motion--ghost() {
-  @include button--disabled();
-
   /* stylelint-disable-next-line declaration-no-important */
   background: transparent !important;
 }
 
-@mixin button--disabled {
+@mixin button--disabled() {
   &:disabled,
   &[aria-disabled] {
+    .m-button__arrow {
```

Comment:
The mixins should be class agnostic

Reply:
i think here makes sense as the icon is hard cobbled in the `template.js`